source/cli.ts

```diff
@@ -30,7 +30,8 @@ const cli = meow(`
 
 			throw new Error(`Found ${diagnostics.length} issues.`);
```

Comment:
With the current implementation, it will print all the error messages and then at the end it will print

> Found 20 issues.

That's not really what we want. Instead, can you remove the `console.log(formatter(diagnostics))` and do `throw new Error(formatter(diagnostics))` and test if that works?